generators/vue/templates/tsconfig.vitest.json.ejs

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.app.json",
+  "exclude": [],
+  "compilerOptions": {
+    "composite": true,
+    "lib": [],
```

Comment:
same?

Reply:
Not sure, this is generated by `npm init create-vue`.
It probably overrides the lib from extended config.